src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/change_orders_status_modal.html.twig

```diff
@@ -0,0 +1,56 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% embed 'PrestaShopBundle:Admin/Helpers:bootstrap_popup.html.twig' with {
+  'id': 'changeOrdersStatusModal',
+  'title': 'Choose an order status'|trans({}, 'Admin.Orderscustomers.Feature'),
+  'closable': true,
+  'actions': [{
+    'type': 'button',
+    'label': 'Update Order Status'|trans({}, 'Admin.Orderscustomers.Feature'),
```

Comment:
What about _Update status_ instead? Do you agree that this will introduce a new behavior since it won't be necessary to go inside an order page to update its status?

Reply:
I've just used the same wording as in legacy, if you think it should be changed, it's :+1: for me.

Reply:
@sarjon, thanks for your quick reply! Am I right about the behavior or is it displayed in another context?

Reply:
It's a legacy page, but the context is the same (change order status in bulk action) in new page as well, except it's displayed in modal. https://prnt.sc/nxujkw

Reply:
Alright. Let's write _Update status_ in this case, localized in _Admin.Orderscustomers.Feature_ as well, please. :-)

Reply:
Done. :+1:

Reply:
Done. :+1: